src/fatality.js

```diff
@@ -1,7 +1,9 @@
 const lockfile = require('./lockfile');
 
 module.exports = err => {
+  console.error('********FATAL********');
```

Comment:
OH YOU.

FWIW I found it hard to read without something calling it out more, especially because it's not that robust right now due to locking being behind a promise. At some point I think it's worth considering writing/using a lockfile library that is synchronous, but it's not a deal-breaker.